src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_slr_is_on_and_transaction_mode_is_none.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability.Retries
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_slr_is_on_and_transaction_mode_is_none : NServiceBusAcceptanceTest
```

Comment:
nitpicking: use enabled instead of on.

Maybe shorten it to `When_slr_enabled_and_transaction_mode_none`?